content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -1,391 +1,287 @@
+    if (string.IsNullOrEmpty(player))
+    {
+        logger.LogInformation("Anonymous player is rolling the dice: {result}", result);
+    }
+    else
+    {
+        logger.LogInformation("{player} is rolling the dice: {result}", player, result);
+    }
 
-<details>
-<summary>View example output</summary>
+    return result.ToString(CultureInfo.InvariantCulture);
+}
 
-```properties
-Activity.TraceId:            54d084eba205a7a39398df4642be8f4a
-Activity.SpanId:             aca5e39a86a17d59
-Activity.TraceFlags:         Recorded
-Activity.ActivitySourceName: Microsoft.AspNetCore
-Activity.DisplayName:        /
-Activity.Kind:               Server
-Activity.StartTime:          2023-02-21T12:19:28.2499974Z
-Activity.Duration:           00:00:00.3106744
-Activity.Tags:
-    net.host.name: localhost
-    net.host.port: 5123
-    http.method: GET
-    http.scheme: http
-    http.target: /
-    http.url: http://localhost:5123/
-    http.flavor: 1.1
-    http.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36 Edg/110.0.1587.50
-    http.status_code: 200
-Resource associated with Activity:
-    service.name: MyService
-    service.instance.id: 2c7ca153-e460-4643-b550-7c08487a4c0c
 ```
 
-</details>
-
-### Manual Instrumentation
-
-Next, add [tracing](/docs/concepts/signals/traces/) via the `System.Diagnostics`
-API.
+In the `Properties` subdirectory, modify a file called `launchSettings.json` and
+replace existing content by the following code:
 
-Paste the following code into your `HomeController`'s `Index` action:
-
-```csharp
-public IActionResult Index()
+```json
 {
-    // Track work inside of the request
-    using var activity = DiagnosticsConfig.ActivitySource.StartActivity("SayHello");
-    activity?.SetTag("foo", 1);
-    activity?.SetTag("bar", "Hello, World!");
-    activity?.SetTag("baz", new int[] { 1, 2, 3 });
-
-    return View();
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "http": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "http://localhost:8080",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
+  }
 }
 ```
 
-When you run the app and navigate to the `/` route, you'll see output about
-[spans](/docs/concepts/signals/traces/#spans) similar to the following:
+Build and run the application with the following command, then open
+<http://localhost:8080/rolldice> in your web browser to ensure it is working.
 
-<details>
-<summary>View example output</summary>
+```sh
+dotnet build
+dotnet run
+```
 
-```text
-Activity.TraceId:            47d25efc8b5e9184ce57e692f5f65465
-Activity.SpanId:             bb864adcf4592f54
+## Instrumentation
+
+Next, you'll use a [OpenTelemetry .NET Automatic Instrumentation](../automatic)
+to instrument the application at launch time. While you can [configure .NET
```

Comment:
would be nice to describe what gets automatically instrumented here.. (the asp.net core server itself, any httpclient calls, etc.. so users know why they are seeing (or not seeing) the telemetry at the end.)

Reply:
I think that it is to early for details. It can go to ../automatic somehow.

I expect form getting started is to present that traces, metrics, and logs can be handled by OTel. Details are not so important.

Reply:
Agreed, we can elaborate more on what's instrumented later. I think it will also be pretty obvious what's instrumented when people run a sample app like in the docs here.